src/harness/harnessIO.ts

```diff
@@ -616,7 +616,11 @@ export namespace Compiler {
             }
         }
 
-        yield [diagnosticSummaryMarker, Utils.removeTestPathPrefixes(minimalDiagnosticsToString(diagnostics, options && options.pretty)) + IO.newLine() + IO.newLine(), diagnostics.length];
+        let topDiagnostics = minimalDiagnosticsToString(diagnostics, options && options.pretty);
+        topDiagnostics = Utils.removeTestPathPrefixes(topDiagnostics);
+        topDiagnostics = topDiagnostics.replace(/^(lib(?:.*)\.d\.ts)\(\d+,\d+\)/igm, "$1(--,--)");

```

Comment:
It might be good idea to create `Utils.removeLibLocations` or something similar and use it everywhere instead ?

Reply:
I wanted to do that, but all of these different locations use entirely different display formats and so need different expressions and replacement strings; I could attempt to write a combined regex but the replacement string would have to be arbitrarily passed in per-replacement.